test/AbstractFactory/ReflectionBasedAbstractFactoryTest.php

```diff
@@ -5,7 +5,7 @@
 namespace LaminasTest\ServiceManager\AbstractFactory;
 
 use ArrayAccess;
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
Note the casing here. One of our phpcs rules for some reason lowercases imports and class names it cannot resolve. You'll need to add an exclusion for that rule so that rewrites like this don't happen.